act-rules-format.md

```diff
@@ -160,26 +160,38 @@ Refer to the [Rule Type](#rule-types) section for detailed definitions of the ru
 Accessibility Requirements Mapping {#accessibility-requirements-mapping}
 ------------------------------------------------------------------------
 
-When an ACT Rule is designed to test conformance to one or more [=Accessibility requirements documents=], the rule <em class="rfc2119">must</em> list all [=accessibility requirements=] from those documents that are not satisfied when one or more of the [=outcomes=] of the rule is `failed`. For example, when designing a rule for WCAG that tests if image buttons have alternative text, the rule maps to success criteria 1.1.1 Non-text content, and 4.1.2 Name, Role, Value. That ACT Rule will list both success criteria in its accessibility requirements mapping.
+When an ACT Rule is designed to test conformance to one or more [=Accessibility requirements documents=], the rule <em class="rfc2119">must</em> list all [=accessibility requirements=] from those documents that are not satisfied when one or more of the [=outcomes=] of the rule is `failed`. The rule <em class="rfc2119">may</em> list accessibility requirements that could be not satisfied when the rule outcome is failed. There are two types of accessibility requirements:
+- [Conformance Requirements](#conformance-requirements)
+- [Secondary Requirements](#secondary-requirements)
 
 Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em> include the following:
 
 1. either the name, title, identifier or summary of the accessibility requirement, and
 2. the name of the [=accessibility requirements document=], and
 3. a link or reference to the [=accessibility requirements document=] if one exists, and
-4. the conformance level associated with the accessibility requirement, if one exists.
-
+4. the conformance level associated with the accessibility requirement, if one exists, and
+5. whether the requirement is a conformance requirement or a secondary requirement.
```

Comment:
You propose "requirement strictness" as a solution in the editors notes but I don't see how that would solve the problems seen in scenarios 2 and 4. I think it could be part of the solution, but could not be the whole solution.

 As I have re-read the scenarios the "rules of thumb" for measuring them seems to be the below for me. I don't think you could wrap 2 and 3 into requirement strictness.  

1. "Requirement Strictness" - Handles scenarios 1 and 3
2. "Passed = satisfied outcome, failed $\neq$ failed outcome" - Scenario 2
3. "Applicable to only some, but not all, passed and failed test cases" - Scenario 4

Reply:
You may be right. I haven't looked into it yet. It's something to discuss I think.